src/current/v24.1/create-security-certificates-openssl.md

```diff
@@ -355,6 +362,10 @@ In the following steps, replace the placeholder text in the code with the actual
     > \q
     ~~~
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/misc/cert-auth-using-x509-subject.md %}
```

Comment:
Do we want this include three times on the same page?

Reply:
I've left the one in the overview and removed the 2 in the body subsections